packages/cactus-plugin-ledger-connector-hedera/.env

```diff
@@ -0,0 +1,2 @@
+MY_ACCOUNT_ID = 0.0.1740
```

Comment:
This should not be checked in, (you just checked in your private account key!).  Instead a `sample.env` file with the names of the env variables and descriptions of their values should be checked in.

But we will also want to align with how other plugins setup their config information.  How they do it should be researched.